setup.cfg

```diff
@@ -1,58 +0,0 @@
-[flake8]
```

Comment:
Just making sure it is intentional to delete this file. I presume that just "taking black defaults" is what you expect to do, including ignoring all of the convenient exceptions? Not refuting it, just making sure that we all know what the intent here is.

Reply:
Intentional.
Only Flake8 remains that doesn't use pyproject.toml. The way I was reading that is they were waiting for a parser to make the standard library (3.11) or for a clear standard to emerge.
Putting flake8 config in it's own file makes it clear (at least to me) that no new configs should be declared in the legacy setup.cfg, preferring pyproject.toml where possible.
Something for the contributing.md, will add.